src/api/agency-client.js

```diff
@@ -0,0 +1,14 @@
+import ApiClient from "./api";
+
+const AgencyClient = {
+  get: function (token) {
+    return ApiClient.get("/api/agency", {
+        headers: {
+            "Authorization": "Bearer " + token,

```

Comment:
Please add request interceptor that will add "Authorization" header before making request.
See here. https://github.com/axios/axios#interceptors